README.asciidoc

```diff
@@ -30,7 +30,13 @@ a| image::images/raspberry_pi_os_lite.png[64bit Lite,200]
 1. Enable passwordless ssh, by copying your public key to the raspberry pi's `~/.ssh/authorized_keys` file. For convenience, you can also use `ssh-copy-id -i $PATH_TO_KEY pi@$PI_IP_ADDRESS`
 1. Ensure you have added the key to your agent with `ssh-add -K $PATH_TO_KEY`. Now try to connect via SSH and ensure you don't get a password prompt.
 
+== Preparations
+The first thing you'll need for the cluster setup is ansible. So please install ansible following the guide here: https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible Installation Guide]
```

Comment:
Is this referring to installing Ansible on the host, or on the raspberry ? Can we clarify?

Reply:
It is for the host as far as I remember

Reply:
Ok, let's clarify:

```suggestion
The first thing you'll need for the cluster setup is to install Ansible on your workstation. Your workstation will connect to each of the raspberry pi nodes via SSH and run all the commands in the Ansible playbook, as you sip your beverage of choice relaxing on your chair. 

You can install Ansible following the guide here: https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible Installation Guide]
```

Reply:
sip you beverage of choice 😆